contents/handbook/engineering/deployment-strategy.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Deployment Strategy
+sidebar: Handbook
+showTitle: true
+---
+
+The way we deploy is a key _product_ decision.
+
+* 'Cloud' = PostHog Cloud, we host it for you in *PostHog's* AWS
+* 'Private Cloud' = You host it, we manage it in your AWS / GCP without needing access to your data
+* 'On Premise' = You host it on your own physical servers / somewhere that PostHog can't maintain it for you remotely
+
```

Comment:
Feels like we're missing 'self-host', where I don't give you access but it's still not a physical server (i.e. it's not on-prem). 

I get the point we're making here, but it's just that on-prem is quite specifically different from me deploying on AWS. 

I think the reason is that these are docs for Enterprise, but in that case we should make it clear.

Reply:
Or maybe we can just have On Premise/Self-Host like Tim had on his table when he showed this